aapcs64/aapcs64.rst

```diff
@@ -944,13 +1009,18 @@ The rules for maintenance of the stack are divided into two parts: a set of cons
 Universal stack constraints
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-At all times the following basic constraints must hold:
+At all times during the execution of a thread T, the following basic
```

Comment:
```suggestion
At all times when a thread T executes, the following basic
```

Reply:
I think “during the execution of X” is the term I'd usually expect to see here.  For example, it occurs quite often in the C++ standard.

Reply:
I'll leave this to your discretion. In Tech Comms style, we're encourage to write in the active voice where possible.